### PR TITLE
fix: Update dns name for issuer/rp adapter

### PR DESCRIPTION
**Pull request by rolsonquadras**
Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: cmd/rp-rest/static/index.html [hunk 1]
```diff
@@ -170,7 +170,7 @@ <h4 class="mt-10 text-lg font-medium tracking-wide uppercase leading-tight">Appl
         <div class="bg-white rounded-lg shadow-lg">
             <div class="px-4 py-8">
                 <button id="creditCardStatement" onclick="getCreditCardStatement()" class="h-24"><img src="img/icon-home-5.png"></button>
-                <h4 class="mt-10 text-lg font-medium tracking-wide uppercase leading-tight">Apply for ?????</h4>
+                <h4 class="mt-10 text-lg font-medium tracking-wide uppercase leading-tight">Apply for New Credit Card</h4>
```
Comment (llorllale) [inline cmd/rp-rest/static/index.html:173]: not sure it makes sense but we can discuss these details later

**Comment by llorllale** [comment]
@rolsonquadras this is more of a fix than a chore, isn't it?

**Comment by rolsonquadras** [comment]
> @rolsonquadras this is more of a fix than a chore, isn't it?

Yes, updated the title/commit.

File: cmd/rp-rest/static/index.html (hunk 1 above)
Comment (rolsonquadras) [reply cmd/rp-rest/static/index.html:173]: yes, better than `?????` for now.

**Pull request merged**